### PR TITLE
Simprints register

### PR DESCRIPTION
**Pull request by mahmud6390**
simprint integration

### CODE REVIEW

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java [hunk 1]
```diff
@@ -124,6 +128,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             onActivityResultListeners.get(requestCode).onActivityResult(requestCode, resultCode, data);
         } else {
             super.onActivityResult(requestCode, resultCode, data);
+            //Get simprint result
+//            if(resultCode == Activity.RESULT_OK && requestCode == JsonFormConstants.ACTIVITY_REQUEST_CODE.REQUEST_CODE_REGISTER){
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java]: Please remove this dead code if you don't intend to use it later.

File: sample/src/main/assets/json.form/basic_form.json [hunk 2]
```diff
@@ -65,6 +65,39 @@
         "type": "choose_image",
         "uploadButtonText": "Take a photo of the child"
       },
+      {
+        "key": "finger_print",
```
Comment (dubdabasoduba) [inline sample/src/main/assets/json.form/basic_form.json]: This two fields have the same key I don't think the updates will work well here.

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by vijay on 24-05-2015.
+ */
+public class FingerPrintViewFactory implements FormWidgetFactory {
+
+    public static int dp2px(Context context, float dp) {
+        Resources r = context.getResources();
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java]: Please name variables accordingly for increased code readability

File: sample/src/main/assets/json.form/basic_form.json (hunk 2 above)
Comment (mahmud6390) [reply sample/src/main/assets/json.form/basic_form.json]: Actually it's sample how we need to add register and verify finger print option. there had a key like "finger_print_option":"register", or "finger_print_option":"verify",

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java (hunk 1 above)
Comment (mahmud6390) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java]: ok will remove. it was just like a snapet how to parse the simprints callback data

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java (hunk 3 above)
Comment (mahmud6390) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java]: ok will change actually was copied from another class ImagePickerViewFactory.

Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java]: No problem. We have gotten the time to update the code standards in all the classes but slowly we shall update them.

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java (hunk 1 above)
Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java]: okay.  I get it now

**Pull request merged**